build/rocm/README.md

```diff
@@ -1,38 +1,209 @@
 
-1.  Install Docker: Follow the [instructions on the docker website](https://docs.docker.com/engine/installation/).
+## 1. Using Prebuilt Docker Images
 
-2. Build a runtime JAX-ROCm docker container and keep this image by running the following command. Note: must pass in appropriate
-options. The example below builds Python 3.12 container.
+The ROCm JAX team provides prebuilt Docker images, which the simplest way to use JAX on ROCm. These images are available on Docker Hub and come with JAX configured for ROCm.
+
+To pull the latest ROCm JAX Docker image, run:
+
+```Bash
+> docker pull rocm/jax-community:latest
+```
+
+Once the image is downloaded, launch a container using the following command:
+
+```Bash
+> docker run -it -d --network=host --device=/dev/kfd --device=/dev/dri --ipc=host --shm-size 64G --group-add video --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -v $(pwd):/jax_dir --name rocm_jax rocm/jax-community:latest /bin/bash
+
+> docker attach rocm_jax
+```
+
+### Notes:
+1. The `--shm-size` parameter allocates shared memory for the container. Adjust it based on your system's resources if needed.
+2. Replace `$(pwd)` with the absolute path to the directory you want to mount inside the container.
+
+***For older versions please review the periodically pushed docker images at:
+[ROCm JAX Community DockerHub](https://hub.docker.com/r/rocm/jax-community/tags).***
+
+### Testing your ROCm environment with JAX:
+
+After launching the container, test whether JAX detects ROCm devices as expected:
+
+```Bash
+> python -c "import jax; print(jax.devices())"
+[RocmDevice(id=0), RocmDevice(id=1), RocmDevice(id=2), RocmDevice(id=3)]
+```
+
+If the setup is successful, the output should list all available ROCm devices.
+
+## 2. Using a ROCm Docker Image and Installing JAX
+
+If you prefer to use the ROCm Ubuntu image or already have a ROCm Ubuntu container, follow these steps to install JAX in the container.
+
+### Step 1: Pull the ROCm Ubuntu Docker Image
+
+For example, use the following command to pull the ROCm Ubuntu image:
+
+```Bash
+> docker pull rocm/dev-ubuntu-22.04:6.3-complete
+```
+
+### Step 2: Launch the Docker Container
+
+After pulling the image, launch a container using this command:
+
+```Bash
+> docker run -it -d --network=host --device=/dev/kfd --device=/dev/dri --ipc=host --shm-size 64G --group-add video --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -v $(pwd):/jax_dir --name rocm_jax rocm/dev-ubuntu-22.04:6.3-complete /bin/bash
+> docker attach rocm_jax
+```
+
+### Step 3: Install the Latest Version of JAX
+
+Inside the running container, install the required version of JAX with ROCm support using pip:
+
+```Bash
+> pip3 install jax[rocm]
+```
+
+### Step 4: Verify the Installed JAX Version
+
+Check whether the correct version of JAX and its ROCm plugins are installed:
```

Comment:
As in the other file, I would avoid referencing specific version numbers here.

Reply:
@superbobry same reason as the above!